src/Zynga/Framework/PgData/V1/PgRow/PkComesFromMemcache.hh

```diff
@@ -114,7 +113,7 @@ abstract class PkComesFromMemcache extends PgRow {
     $connectionString = '';
 
     $tableName = $this->getTableName();
-    $pkKey = 'pgd:'.md5($connectionString).'|'.$tableName.':pk';
+    $pkKey = 'pgd:'.hash("sha256", $connectionString).'|'.$tableName.':pk';
     return $pkKey;
```

Comment:
$connectionString is empty/missed here. Is that supposed to be write db connection string?